clients/fidesui/src/index.ts

```diff
@@ -53,7 +53,7 @@ export * from "@chakra-ui/visually-hidden";
  */
 /* eslint-disable import/export */
 export * from "./icons";
-export { AddIcon, QuestionIcon, WarningIcon, LinkIcon } from "./icons";
+export { AddIcon, LinkIcon,QuestionIcon, WarningIcon } from "./icons";
```

Comment:
as nit picky as it gets, but looks like this could use an extra space after the `,`

Reply:
ah nice!